README.md

```diff
@@ -67,11 +67,11 @@ The **Formatter**, is in charge of preparing the message to be sent in our notif
 The template or 'format' to be used should be included in the use case configurations, which we'll review in a further step. It's
 implementation can be found under `/bas/formatter/birthday.rb`.
 
-### 4. Dispatcher - Sending your notification
+### 4. Process - Optional Data Process
 
-Finally, the **Dispatcher** basically, sends or dispatches the formatted message into a destination, since the use case was implemented for
+Finally, the **Processor** basically, allow required data process depending on the use case like sending formatted messages into a destination. In this case, since the use case was implemented for
```

Comment:
Hello @L-Zuluaga, yes, you are right. I was thinking of referring to the component in charge of the "Process" the "Processor" but it is better to write directly the name of the component to avoid confusion.